tests/display.rs

```diff
@@ -350,5 +350,5 @@ fn debug_string() {
     use glium::backend::Facade;
 
     let display = support::build_display();
-    display.get_context().insert_debug_marker("Hello world");
+    display.get_context().insert_debug_marker("Hello world").unwrap();
```

Comment:
The test is checking that the function exists and doesn't change its API.
You can't predict whether or not it is supposed to succeed, as it depends on the hardware and configuration of the user.